baybe/utils/plotting.py

```diff
@@ -0,0 +1,64 @@
+"""Utilities that are useful for plotting."""
+
+import json
+import os
+import sys
+import warnings
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+
+def create_plots(data: pd.DataFrame, name: str, **kwargs) -> None:
```

Comment:
Instead of putting a str-name here, a Path would be a better option (also, this would make the explanation in the docstring below redundant)

Reply:
I changed this such that it requires a `path` (for the location) and a `base_name` (for the name of the actual plots). Reason for this was that we append stuff like "_check", "_light" and "_dark", and doing this when only being provided a `Path` is a bit messy, since this path then needs to carry the information of the name of the file - and this information then needs to be extracted.
Also, there were (as usual....) problems with the duality of executing the files as pure python files and as jupyter notebooks, and having these two aspects cleanly separated makes these issues significantly less painful.